### ISSUE TITLE
Inconsistent handling of groups with roles assigned to them through memberships

### ISSUE THREAD

**Issue by samuelharmer** [maxtepkeev/python-redmine#220]
I have been experiencing a strange bug with my code and have tracked it down to inconsistent handling of resources retrieved via `project.memberships`.

```
proj = redmine.project.get('citadel')
proj.memberships[0].group
Out[1]: {'id': 1, 'name': 'Council of Ricks'}
proj.memberships[1].user
Out[7]: <redminelib.resources.User #3 "Maximums Rickimus">
```

I would expect `.group` to be of type `redminelib.resources.Group` rather than a `dict`.

I'm assuming it's something to do with the way [`class ProjectMembership`](https://github.com/maxtepkeev/python-redmine/blob/08c0b6466bfe93110467bb8dae9bd7d42b2db3c7/redminelib/resources/standard.py#L321) doesn't include any reference to groups. Perhaps this is as simple as changing [L336](https://github.com/maxtepkeev/python-redmine/blob/08c0b6466bfe93110467bb8dae9bd7d42b2db3c7/redminelib/resources/standard.py#L336) from:
```
    _resource_map = {'project': 'Project', 'user': 'User'}
```
to:
```
    _resource_map = {'project': 'Project', 'user': 'User', 'group': 'Group'}
```

**Comment by samuelharmer** [comment]
> Perhaps this is as simple as changing [L336](https://github.com/maxtepkeev/python-redmine/blob/08c0b6466bfe93110467bb8dae9bd7d42b2db3c7/redminelib/resources/standard.py#L336) from:
> 
> ```
>     _resource_map = {'project': 'Project', 'user': 'User'}
> ```
> to:
> 
> ```
>     _resource_map = {'project': 'Project', 'user': 'User', 'group': 'Group'}
> ```

Yep; that seems to do it.

Are you happy to fix this?

**Comment by maxtepkeev** [comment]
@samuelharmer Thanks for the bug report. This is fixed in d8aaa3207ae4002af9071ced411b48ec85a3bf16.

Don't have time to do a release right now, but will do next week.
